pkg/netpol/connlist/exposed_pods.go

```diff
@@ -6,21 +6,22 @@ import "github.com/np-guard/netpol-analyzer/pkg/netpol/internal/common"
 type ExposedPeer interface {
 	// ExposedPeer is a peer for which the analysis found some potential exposure info
 	ExposedPeer() Peer
-	// IngressExposure list of the potential Ingress connections to the ExposedPeer
+	// IngressExposure is a list of the potential Ingress connections to the ExposedPeer
 	IngressExposure() []XgressExposureData
-	// EgressExposure list of the potential Egress connections from the ExposedPeer
+	// EgressExposure is a list of the potential Egress connections from the ExposedPeer
 	EgressExposure() []XgressExposureData
 }
 
-// XgressExposureData data of potential connectivity for an existing peer to/from a representative peer
+// XgressExposureData contains data of potential connectivity for an existing peer to/from a representative peer
```

Comment:
since this is the interface we should not use the internal term `representative peer` in the documentation here as well..

Reply:
changed